packages/graphql/src/translate/translate-top-level-cypher.ts

```diff
@@ -163,10 +163,15 @@ export function translateTopLevelCypher({
         });
         cypherStrs.push(...cypherStatement);
     } else {
+        const columnName = field.columnName;
         cypherStrs.push(`
-            CALL apoc.cypher.doIt("${statement}", ${apocParamsStr}) YIELD value
-            WITH [k in keys(value) | value[k]][0] AS this
-            `);
+            CALL {
+                WITH *
```

Comment:
This is a bit of a weirdness, we shouldn't really need `WITH *` (or anything for that matter) because it is a top-level Mutation?

Reply:
Good spot